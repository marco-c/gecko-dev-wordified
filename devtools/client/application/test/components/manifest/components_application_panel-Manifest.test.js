/
*
Any
copyright
is
dedicated
to
the
Public
Domain
.
http
:
/
/
creativecommons
.
org
/
publicdomain
/
zero
/
1
.
0
/
*
/
"
use
strict
"
;
/
/
Import
libs
const
{
shallow
}
=
require
(
"
enzyme
"
)
;
const
{
createFactory
}
=
require
(
"
react
"
)
;
const
Manifest
=
createFactory
(
require
(
"
devtools
/
client
/
application
/
src
/
components
/
manifest
/
Manifest
"
)
)
;
const
{
MANIFEST_SIMPLE
}
=
require
(
"
.
.
/
fixtures
/
data
/
constants
"
)
;
/
*
*
Test
for
Manifest
component
*
/
describe
(
"
Manifest
"
(
)
=
>
{
it
(
"
renders
the
expected
snapshot
"
(
)
=
>
{
const
wrapper
=
shallow
(
Manifest
(
MANIFEST_SIMPLE
)
)
;
expect
(
wrapper
)
.
toMatchSnapshot
(
)
;
}
)
;
}
)
;
