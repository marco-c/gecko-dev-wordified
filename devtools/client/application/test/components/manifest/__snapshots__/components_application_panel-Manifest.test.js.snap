/
/
Jest
Snapshot
v1
https
:
/
/
goo
.
gl
/
fbAQLP
exports
[
Manifest
renders
the
expected
snapshot
1
]
=
<
article
>
<
Localized
id
=
"
manifest
-
view
-
header
"
>
<
h1
className
=
"
app
-
page__title
"
/
>
<
/
Localized
>
<
ManifestSection
key
=
"
manifest
-
section
-
0
"
title
=
"
manifest
-
item
-
warnings
"
>
<
ManifestItemWarning
key
=
"
warning
-
0
"
warning
=
{
Object
{
"
warn
"
:
"
This
is
a
warning
"
}
}
/
>
<
/
ManifestSection
>
<
ManifestSection
key
=
"
manifest
-
section
-
1
"
title
=
"
manifest
-
item
-
identity
"
>
<
ManifestItem
key
=
"
name
"
label
=
"
name
"
>
foo
<
/
ManifestItem
>
<
/
ManifestSection
>
<
ManifestSection
key
=
"
manifest
-
section
-
2
"
title
=
"
manifest
-
item
-
presentation
"
>
<
ManifestItem
key
=
"
lorem
"
label
=
"
lorem
"
>
ipsum
<
/
ManifestItem
>
<
ManifestItem
key
=
"
foo
"
label
=
"
foo
"
>
bar
<
/
ManifestItem
>
<
/
ManifestSection
>
<
ManifestSection
key
=
"
manifest
-
section
-
3
"
title
=
"
manifest
-
item
-
icons
"
>
<
ManifestItem
key
=
"
1x1
"
label
=
"
1x1
"
>
something
.
png
<
/
ManifestItem
>
<
/
ManifestSection
>
<
/
article
>
;
